src/Bem2d.jl

```diff
@@ -147,22 +211,13 @@ end
 
 # Calculate displacements and stress for constant slip/traction elements
 export constdispstress
-function constdispstress(fun2dispstress, x, y, els, idx, xcomp, ycomp, mu, nu)
-    disp, stress = zeros(length(x), 2), zeros(length(x), 3)
-    _x, _y = zeros(length(x)), zeros(length(x))
-    _xcomp, _ycomp = zeros(1), zeros(1)
-    f = zeros(length(x), 7)
-    _disp, _stress = zeros(length(x), 2), zeros(length(x), 3)
-    @inbounds @simd for j in 1:length(idx)
```

Comment:
`@simd` really only has an effect on the very innermost loops where you're down to pure arithmetic.